src/app/static/src/app/store/colourScales/colourScales.ts

```diff
@@ -0,0 +1,52 @@
+import {Dict} from "../../types";
+import {localStorageManager} from "../../localStorageManager";
+import {Module} from "vuex";
+import {RootState} from "../../root";
+import {getters} from "./getters";
+import {actions} from "./actions";
+import {mutations} from "./mutations";
+
+export interface ColourScalesState {
```

Comment:
It could do, though as I said above wouldn't really be consistent with the other structure of plottingSelections (where bubble and choro have their own selections), so these would need to be top level properties on plottingSelections anyway, so I thought it might be cleaner as a new module. Happy to combine if you'd prefer.

Reply:
My feeling is it's cleaner as one module, but since we're split on this and you've already written the code I'll leave it up to you!

Reply:
Fair enough, you've talked me into it! :)